org.openscad.OpenSCAD.yaml

```diff
@@ -3,8 +3,10 @@ runtime: org.kde.Platform
 runtime-version: 5.15-23.08
 sdk: org.kde.Sdk
 command: openscad
-rename-desktop-file: openscad-nightly.desktop
 rename-icon: openscad-nightly
+rename-mime-file: openscad.xml
+rename-desktop-file: openscad.desktop
+rename-appdata-file: openscad.appdata.xml
```

Comment:
according to this https://github.com/openscad/openscad/blob/master/CMakeLists.txt#L1315 the appstream file is already renamed to something else.

Reply:
Right, I thought I'm looking at those renames in the logs, but I guess instead it was that `make install` instead... tbc... at some later time. Thanks!